pettingzoo/mpe/_mpe_utils/simple_env.py

```diff
@@ -231,75 +246,80 @@ def step(self, action):
         self._cumulative_rewards[cur_agent] = 0
         self._accumulate_rewards()
 
-    def render(self, mode="human"):
-        from . import rendering
-
-        if self.viewer is None:
-            self.viewer = rendering.Viewer(700, 700)
-
-        # create rendering geometry
-        if self.render_geoms is None:
-            # import rendering only if we need it (and don't import for headless machines)
-            # from gym.envs.classic_control import rendering
-            # from multiagent._mpe_utils import rendering
-            self.render_geoms = []
-            self.render_geoms_xform = []
-            for entity in self.world.entities:
-                geom = rendering.make_circle(entity.size)
-                xform = rendering.Transform()
-                if "agent" in entity.name:
-                    geom.set_color(*entity.color[:3], alpha=0.5)
-                else:
-                    geom.set_color(*entity.color[:3])
-                geom.add_attr(xform)
-                self.render_geoms.append(geom)
-                self.render_geoms_xform.append(xform)
-
-            # add geoms to viewer
-            self.viewer.geoms = []
-            for geom in self.render_geoms:
-                self.viewer.add_geom(geom)
-
-            self.viewer.text_lines = []
-            idx = 0
-            for agent in self.world.agents:
-                if not agent.silent:
-                    tline = rendering.TextLine(self.viewer.window, idx)
-                    self.viewer.text_lines.append(tline)
-                    idx += 1
-
-        alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
-        for idx, other in enumerate(self.world.agents):
-            if other.silent:
-                continue
-            if np.all(other.state.c == 0):
-                word = "_"
-            elif self.continuous_actions:
-                word = "[" + ",".join([f"{comm:.2f}" for comm in other.state.c]) + "]"
-            else:
-                word = alphabet[np.argmax(other.state.c)]
+    def enable_render(self):
+        self.screen = pygame.display.set_mode(self.screen.get_size())
+        self.renderOn = True
+        self.draw()
 
-            message = other.name + " sends " + word + "   "
+    def render(self, mode="human"):
+        if not self.renderOn and mode == "human":
+            # sets self.renderOn to true and initializes display
+            self.enable_render()
+        elif mode == "human":
+            self.draw()
+
+        observation = np.array(pygame.surfarray.pixels3d(self.screen))
+        if mode == "human":
+            pygame.display.flip()
```

Comment:
Hi there,

Easy and well written code. I could read really quick.

I would suggest small changes to improve a little bit readability, really small changes.

```suggestion
        self.enable_render()

        observation = np.array(pygame.surfarray.pixels3d(self.screen))
        if mode == "human":
            self.draw()
            pygame.display.flip()
```

Reply:
See reply below, same thoughts apply.